src/CookBook/CookBook.App/ViewModels/Ingredient/IngredientListViewModel.cs

```diff
@@ -25,23 +31,27 @@ protected override async Task LoadDataAsync()
     [RelayCommand]
     private async Task GoToCreateAsync()
     {
-        await navigationService.GoToAsync("/edit");
+        await navigationService.GoToAsync(NavigationService.IngredientEditRouteRelative);
     }
 
     [RelayCommand]
     private async Task GoToDetailAsync(Guid id)
     {
-        await navigationService.GoToAsync<IngredientDetailViewModel>(
-            new Dictionary<string, object?> { [nameof(IngredientDetailViewModel.Id)] = id });
+        await navigationService.GoToAsync(NavigationService.IngredientDetailRouteRelative,
+            new Dictionary<string, object?>
+            {
+                [nameof(IngredientDetailViewModel.Id)] = id
+            }
+        );
     }
 
-    public async void Receive(IngredientEditMessage message)
+    public void Receive(IngredientEditMessage message)
     {
-        await LoadDataAsync();
+        ForceDataRefresh = true;
```

Comment:
Nadávalo by zde smysl udělat na bázi metodu, které se postará o nastavení `ForceDataRefresh = true;`, např. `ForceDataRefreshOnNextAppearing`?

Reply:
Myslíš v rámci encapsulation? Akože nepríde mi, že by to niečo zásadné prinášalo. Ale akože kľudne ak chceš.

Reply:
Možná je to jen můj pocit, ale přijde mi intuitivnější použít volání metody, když chci, aby se buď teď, nebo v budoucnu něco stalo. Uložení si informace, že to je třeba si překreslit jsem bral, že se spíše implementační detail báze.

Nemám ale silné preference. Pokud je tohle běžný způsob jak se to v MAUI dělá tak klidně to takto ponech :).